src/apps/bristleback_apps/pme_do_sensor/user_code/user_code.cpp

```diff
@@ -204,8 +208,9 @@ void loop(void) {
       size_t encodedLen = 0;
       if (PmeDissolvedOxygenMsg::encode(d, cborBuf, sizeof(cborBuf), &encodedLen) ==
           CborNoError) {
-        printf("#  DOT Encoding success! | Topic: %s, cborBuf: %d, \n", pmeDoTopic,
-               cborBuf);
+        printf("#  DOT Encoding success! | Topic: %s, cborBuf: %" PRIx8 "%" PRIx8 "%" PRIx8
+               "...\n",
+               pmeDoTopic, cborBuf[0], cborBuf[1], cborBuf[2]);
```

Comment:
This printing of the pointer as an integer was not at all valid. I decided to keep it simple and just print the first few bytes. The cbor buffer will always be long enough.